kuma/wiki/tests/test_forms.py

```diff
@@ -179,51 +181,51 @@ def test_case_sensitive_tags(self, mock_requests):
         self.assertTrue(rev_form.is_valid())
         self.assertEqual(rev_form.cleaned_data['tags'], '"JavaScript"')
 
-    @pytest.mark.spam
-    @requests_mock.mock()
-    def test_akismet_enabled(self, mock_requests):
+    def _setup_akismet_post(
```

Comment:
I was following the convention of the existing ``_test_akismet_error`` method. I assumed that was named that way to say "this is a helper method, not a test method", but I guess it could have been "hey py.test, pay no attention to this method".

How about I rename ``setup_akismet_post`` and move it under ``tearDown`` ?